core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -71,6 +76,14 @@ object Http4sOrgPlugin extends AutoPlugin {
       skipIfIrrelevant(unusedCompileDependenciesTest)
     )
 
+  lazy val scalafixSettings: Seq[Setting[_]] =
+    Seq(
+      scalafixDependencies ++= Seq(
+        "org.http4s" %% "http4s-scalafix-internal" % "0.23.12",
+        "com.github.liancheng" %% "organize-imports" % "0.6.0"
+      )
+    )
```

Comment:
Weird or no?

Actually I think we can upstream many/all of the internal rules to [typelevel-scalafix](https://github.com/typelevel/typelevel-scalafix). They seem widely applicable.

Reply:
"Upstream to Typelevel unless it's HTTP" has been my mantra for many things.  Seems applicable here, too.